### PR TITLE
release: 0.0.1-alpha2

### PR DESCRIPTION
**Pull request by PabloReszczynski**
Prepare the release to 0.0.1-alpha2

Still in alpha until we can ensure that the module can be imported succesfully from unpkg.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,12 +27,18 @@
   ],
   "homepage": "https://github.com/Topsort/banners.js#readme",
   "exports": {
-    ".": "./dist/banners.esm.js"
+    ".": {
+      "import": "./dist/banners.mjs",
+      "types": "./types/index.d.ts"
```
Comment (jbergstroem) [inline package.json:32]: This is missing `error.d.ts`. You could merge them I guess?

Comment (PabloReszczynski) [reply package.json:32]: The `types` directory is generated by vite, so I don't know how I could do that

Comment (PabloReszczynski) [reply package.json:32]: I just changed it to an array

Comment (jbergstroem) [reply package.json:32]: Few options:
- you can use the plugin I suggested and pass an option that basically merges the types into one file
- you can create a type entrypoint that imports the others
- keep errors out of it for now

**Review by jbergstroem — APPROVED** [review]
I'd like to see us importing `skeleton-webcomponent-loader` as part of the package - I think it would make a better example - but we can do this at a later stage.

File: package.json (hunk 1 above)
Comment (PabloReszczynski) [reply package.json:32]: using vite-plugin-dts i'm not able :s

Comment (PabloReszczynski) [reply package.json:26]: okay, got it

**Pull request merged**